.bazelrc_shared

```diff
@@ -1,44 +1,54 @@
-common --announce_rc
-common --color=yes
-# TODO: remove this when we move to Bazel 8. This is the default in Bazel 8.
-common --experimental_remote_cache_eviction_retries=5
-
-# TODO: enable path mapping when we move to Bazel 8. There is a bug in 7. See:
-# https://github.com/bazelbuild/bazel/issues/23990
-# In the meantime, if you need to test path mapping you should disable multiplex
-# workers with the following build option:
-#build --noworker_multiplex
-#common --experimental_output_paths=strip
-#common --modify_execution_info=CppCompile=+supports-path-mapping
-
-build --experimental_strict_action_env
-build --javacopt="-source 21 -target 21"
+# Java options
 build --java_language_version="21"
 build --java_runtime_version="remotejdk_21"
+build --javacopt="-source 21 -target 21"
 build --tool_java_language_version="21"
 build --tool_java_runtime_version="remotejdk_21"
 
+# Other options
+build --experimental_use_hermetic_linux_sandbox
+build --experimental_worker_cancellation
+build --experimental_worker_multiplex_sandboxing
```

Comment:
Looks like the CI build is failing because of the multiplex sandboxing bug. Which suggests that bug is either a bug in our rule set or still present in Bazel 8. We'll want to disable this for now.

Reply:
Yup, I'll re-disable this.